src/App.jsx

```diff
@@ -52,7 +52,7 @@ const App = () => {
         <Home
           setActors={setActors}
           actors={actors}
-          key={'home'}
+          key={'Home'}
```

Comment:
Nit: Since this isn't a variable this could probably just be `key=Home`

Reply:
Good catch, thank you. Fixing in upcoming commit.